### ISSUE TITLE
Feature request: Support for resource "Kinesis video streams"

### ISSUE THREAD

**Issue by davesade** [terraform-providers/terraform-provider-aws#2994]
Hello!
I think it would be wonderful to have also Kinesis video streams supported in the provider. I couldn't find it in your issue list, so I'd like to request it. Something like this:

```
resource "aws_kinesis_videostream" "test" {
  name             = "KinesisVideostream"
  retention_period = 48
  media_type = "h.264"
  encryption_type = "KMS"
  kms_key_id = "arn::..."

  shard_level_metrics = [
    "IncomingBytes",
    "OutgoingBytes",
  ]

  tags {
    Name = "Test video stream" 
  }
}

```

**Comment by apparentlymart** [comment]
Hi @davesade! Thanks for this feature request.

The Terraform team at HashiCorp won't be able to work on this in the near future due to our focus being elsewhere, but we'd be happy to review a pull request if you or someone else has the time and motivation to implement it. I think under our usual naming conventions it would be best named `aws_kinesis_video_stream`, since AWS seems to spell it with a space between those two words in the product documentation.

Alternatively, if others would also like to see this implemented I'd encourage adding a :+1: upvote reaction to the original issue comment (not to _this_ comment), which we use as one of the inputs to prioritize work for the Terraform team.

**Comment by bflad** [comment]
A new `aws_kinesis_video_stream` resource has been merged and will release with version 2.54.0 of the Terraform AWS Provider, late next week. Thanks to @teraken0509 for the implementation. 🎉
